typescript/web/src/components/labeling-tool/drawing-tool-bar/drawing-tool.tsx

```diff
@@ -371,6 +392,20 @@ export const DrawingTool = () => {
                   <ChakraIoColorWandOutline size="1.3em" />
                 </Box>
               </ToolSelectionPopoverItem>
+              <ToolSelectionPopoverItem
```

Comment:
@nicOwlas The tool appears after Auto-polygon which I find somewhat counter-intuitive as it's a manual tool. Shouldn't we place it above it?

Reply:
Yes